src/resmom_win/start_exec.c

```diff
@@ -1997,8 +1998,11 @@ finish_exec(job *pjob)
 			/* Need to launch a shell*/
 			launch_shell = 1;
 		}
+		/* If we fail to get cmd shell(unlikely), use "cmd.exe" as shell */
+		if (0 != get_cmd_shell(cmd_shell, sizeof(cmd_shell)))
```

Comment:
I have a habit of putting the constant first in a variable comparison since it compels the  compiler to flag an error on accidental assignment = in place of ==(equal to).  For consistency, I have adopted this for all comparisons. I can change it as you suggested.